src/components/List.spec.js

```diff
@@ -0,0 +1,21 @@
+import { toBeChecked } from '@testing-library/jest-dom/dist/matchers';
+import { render, screen } from '@testing-library/react';
+import List from './List.js';
+
+describe('List', () => {
+  it('renders the list', () => {
+    render(<List />);
+
+    const list = screen.getByRole('list');
+
+    expect(list).toBeInTheDocument();
+  });
+
+  it('renders all ten list items', () => {
+    render(<List />);
+
+    const listItems = screen.getAllByRole('listitem');
+
+    expect(listItems.length).toBe(10);
```

Comment:
Instead of `expect(listItems.length).toBe(10)` you can use `toHaveLength`. Example: `expect(lisItems).toHaveLength(10)`